select-xamarin-sdk-v2.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e -o pipefail
+
+echo "Set SDK"
+
+get_framework_path() {
+  local framework=$1
+  case $framework in
+  mono) echo '/Library/Frameworks/Mono.framework/Versions' ;;
+  ios) echo '/Library/Frameworks/Xamarin.iOS.framework/Versions' ;;
+  android) echo '/Library/Frameworks/Xamarin.Android.framework/Versions' ;;
+  mac) echo '/Library/Frameworks/Xamarin.Mac.framework/Versions' ;;
+  *) ;;
+  esac
+}
+
+set_current_folder() {
+  local framework=$1
+  local version=$2
+
+  echo "framework = $framework, version = $version"
+
+  if [ ! -z ${framework} ]; then
+    local countDigit=$(echo "${version}" | grep -o "\." | grep -c "\.")
+    if [[ ! countDigit -eq 1 ]]; then
+      echo "Wrong framework's versions."
+      return
+    fi
+  fi
+
+  local FOLDER=$(get_framework_path "$framework")
+  sudo rm -f ${FOLDER}/Current
+  sudo ln -s "${FOLDER}/${version}" "${FOLDER}/Current"
+}
+
+for arg in "$@"; do
+  key=$(echo $arg | cut -f1 -d=)
+  value=$(echo $arg | cut -f2 -d=)
+
+  case $key in
+  mono) set_current_folder $key $value ;;
+  ios) set_current_folder $key $value ;;
+  android) set_current_folder $key $value ;;
+  mac) set_current_folder $key $value ;;
```

Comment:
Since the action is the same for all keys, may be we can use
```
case $key in
  mono|ios|android|mac) set_current_folder $key $value ;;
  *) ;;
```

See more examples in https://linuxize.com/post/bash-case-statement/#case-statement-example